flake.nix

```diff
@@ -468,7 +468,37 @@
               '';
             };
 
+            frontend-static = let bp = pkgs.callPackage npm-buildpackage { };
+            in bp.buildYarnPackage {
+              nativeBuildInputs = [ pkgs.pkg-config pkgs.vips pkgs.python3 ];
+              src = ./frontend;
+              yarnBuildMore = "yarn export";
```

Comment:
just double check, there was not special yarn nix pacakges? so we use npm. i have not google so.

Reply:
No we're using the yarn.lock now combined with serokell's buildYarnPackage which should have a good enough reproducibility guarantee as i believe it stores hashes of all of the dependencies. It is true that there are also node packages in nixpkgs,which is an option we can consider in the future, but probably not what we want because we want to still let our fe devs be able to use `yarn` directly.